### DISCUSSION TITLE
Option for same short url for a long url and use of message broker

### DISCUSSION THREAD

**Discussion by yatharth2307** [thedevs-network/kutt#497] [Ideas]
Hi there, really grateful to the kutt team for creating such a wonderful tool.  I will be handling a huge load of traffic so want to use some message broker for asynchronous calls. 
I'm willing to contribute myself to have an option for the same. Please let me know what do you think.

**Comment by poeti8** [comment]
Sounds like an idea. A proposal on what you want you do and does that impact and benefits the system would be a start.
